### ISSUE TITLE
Implement Scavenging

### ISSUE THREAD

**Issue by jpatton-USGS** [usgs/Glass2#33]
Implement GlassLib Method(s) for scavenging picks from existing detections.

**Comment by mguy-usgs** [comment]
looks like serious progress, did you get code from Caryl already?

On Wed, Jul 8, 2015 at 12:46 PM, John Patton <notifications@github.com>
wrote:

> Implement GlassLib Method(s) for scavenging picks from existing detections.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/usgs/Glass2/issues/33>.
>



-- 
Michelle Guy
Computer Scientist
NEIC/GHSC/USGS
Golden, CO
303-273-8650
mguy@usgs.gov

**Comment by jpatton-USGS** [comment]
No, I'm setting up all the contract tasks as issues in github so that we
can (theoretically) track progress when she gets going.  Check out
https://waffle.io/usgs/Glass2

John

John Patton
IT Specialist
Geologic Hazards Science Center
U.S. Geological Survey
303.273.8416 (office)
jpatton@usgs.gov

**Comment by mguy-usgs** [comment]
oh, OK, it nice to have the list there.
thanks.

On Thu, Jul 9, 2015 at 8:36 AM, John Patton <notifications@github.com>
wrote:

> No, I'm setting up all the contract tasks as issues in github so that we
> can (theoretically) track progress when she gets going. Check out
> https://waffle.io/usgs/Glass2
>
> John
>
> John Patton
> IT Specialist
> Geologic Hazards Science Center
> U.S. Geological Survey
> 303.273.8416 (office)
> jpatton@usgs.gov
>
> On Thu, Jul 9, 2015 at 8:34 AM, Michelle <notifications@github.com> wrote:
>
> > looks like serious progress, did you get code from Caryl already?
> >
> > On Wed, Jul 8, 2015 at 12:46 PM, John Patton <notifications@github.com>
> > wrote:
> >
> > > Implement GlassLib Method(s) for scavenging picks from existing
> > detections.
> > >
> > > —
> > > Reply to this email directly or view it on GitHub
> > > <https://github.com/usgs/Glass2/issues/33>.
> > >
> >
> >
> >
> > --
> > Michelle Guy
> > Computer Scientist
> > NEIC/GHSC/USGS
> > Golden, CO
> > 303-273-8650
> > mguy@usgs.gov
> >
> > —
> > Reply to this email directly or view it on GitHub
> > <https://github.com/usgs/Glass2/issues/33#issuecomment-120003617>.
>
> >
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/usgs/Glass2/issues/33#issuecomment-120005474>.
>



-- 
Michelle Guy
Computer Scientist
NEIC/GHSC/USGS
Golden, CO
303-273-8650
mguy@usgs.gov
